skimage/measure/simple_metrics.py

```diff
@@ -142,3 +144,100 @@ def compare_psnr(im_true, im_test, data_range=None):
 
     err = compare_mse(im_true, im_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def _pad_to(arr, shape):
+    """Pad an array with trailing zeros to a given target shape.
+
+    Parameters
+    ----------
+    arr : ndarray
+        The input array.
+    shape : tuple
+        The target shape.
+
+    Returns
+    -------
+    padded : ndarray
+        The padded array.
+
+    Examples
+    --------
+    >>> _pad_to(np.ones((1, 1), dtype=int), (1, 3))
+    array([[1, 0, 0]])
+    """
+    assert all(s >= i for s, i in zip(shape, arr.shape)),\
+        "Target shape must be strictly greater than input shape."
+    padding = [(0, s-i) for s, i in zip(shape, arr.shape)]
+    return np.pad(arr, pad_width=padding, mode='constant', constant_values=0)
+
+
+def compare_nmi(im_true, im_test, *, bins=100):
+    """Compute the normalized mutual information.
```

Comment:
The handful of Travis build failures are due to the formatting in this docstring:

> Warning, treated as error:
> /home/travis/venv/lib/python3.6/site-packages/skimage/measure/simple_metrics.py:docstring of skimage.measure.compare_nmi:9:Block quote ends without a blank line; unexpected unindent.
make[1]: *** [html] Error 2

Commenting here to try and save you a bit of time. [Build logs here.](https://travis-ci.org/scikit-image/scikit-image/builds/532135654?utm_source=github_status&utm_medium=notification)